.github/workflows/e2e-windows.yml

```diff
@@ -95,7 +95,7 @@ jobs:
 
     - name: ⚙️ Setup (get latest bins and configs and decode fixtures)
       working-directory: C:/cardano-wallet/test/e2e
-      run: rake setup[%NETWORK%,%PR%]
```

Comment:
I've noticed a difference that in this file we use `%` but in adjacent workflows for linux and macos the `$` is used.

Reply:
That's how you specify env variables on Windows.